src/Form/ThematicCommunityMembershipFilterType.php

```diff
@@ -0,0 +1,170 @@
+                'choices' => [
+                    'common.all' => null,
+                    'common.adherent.subscribed' => true,
+                    'common.adherent.unsubscribed' => false,
+                ],
+                'choice_value' => function ($choice) {
+                    return false === $choice ? '0' : (string) $choice;
+                },
+            ])
+            ->add('smsSubscription', ChoiceType::class, [
+                'required' => false,
+                'choices' => [
+                    'common.all' => null,
+                    'common.adherent.subscribed' => true,
+                    'common.adherent.unsubscribed' => false,
+                ],
+                'choice_value' => function ($choice) {
+                    return false === $choice ? '0' : (string) $choice;
+                },
+            ])
+            ->add('joinedSince', DateType::class, ['required' => false, 'widget' => 'single_text', 'html5' => true])
+            ->add('joinedUntil', DateType::class, ['required' => false, 'widget' => 'single_text', 'html5' => true])
+            ->add('motivation', ChoiceType::class, [
+                'required' => false,
+                'choices' => ThematicCommunityMembership::MOTIVATIONS,
+                'choice_label' => function (string $choice) {
+                    return 'admin.thematic_community.membership.motivations.'.$choice;
+                },
+            ])
+            ->add('categories', EntityType::class, [
+                'required' => false,
+                'multiple' => true,
+                'class' => UserListDefinition::class,
+                'query_builder' => function (EntityRepository $er) {
+                    return $er->createQueryBuilder('uld')
+                        ->where('uld.type IN (:tc_types)')
+                        ->setParameter('tc_types', UserListDefinitionEnum::THEMATIC_COMMUNITY_CODES)
+                    ;
+                },
+            ])
+            ->add('expert', ChoiceType::class, [
+                'required' => false,
+                'expanded' => false,
+                'multiple' => false,
+                'placeholder' => 'Tous',
+                'choices' => [
+                    'Non' => 0,
+                    'Oui' => 1,
+                ],
+            ])
+            ->add('with_job', ChoiceType::class, [
+                'required' => false,
+                'expanded' => false,
+                'multiple' => false,
+                'placeholder' => 'Tous',
+                'choices' => [
+                    'Non' => 0,
+                    'Oui' => 1,
+                ],
+            ])
+            ->add('job', ChoiceType::class, [
+                'required' => false,
+                'expanded' => false,
+                'multiple' => true,
+                'choices' => JobEnum::JOBS,
+                'choice_label' => function (string $choice) {
+                    return $choice;
```

Comment:
same here, it's the default behavior

Reply:
no, as the array only have values, if I don't add this, the choice displayed to the user will be `0, 1, 2, ...`